### ISSUE TITLE
Skeleton not working on Windows

### ISSUE THREAD

**Issue by Netzbewegung-Backend** [FriendsOfTYPO3/content-blocks#319]
**Describe the bug**
Using skeleton directory (`content-blocks-skeleton`) does not work on Windows (10) Systems. Nothing is copied from the Skeleton.

```
$skeletonPath : D:\vhosts\example.local/content-blocks-skeleton/content-element
```

**To Reproduce**
1. Have a Windows Operating System
2. Create Skeleton Directory with a file: content-blocks-skeleton/content-element/templates/backend-preview.html
3. Excecut vendor/bin/typo3 content-blocks:create --content-type="content-element"
4. Check if backend-preview.html from Skeleton is used in new content block

**Expected behavior**
Skeleton should also work on Windows

**TYPO3 Version** 13.4.2

**Content Blocks Version:** 1.1.0

**Additional context**
As a quick hotfix i tested with: `GeneralUtility::fixWindowsFilePath($skeletonPath)`, then it works as expected:
```
\TYPO3\CMS\ContentBlocks\Builder\ContentBlockBuilder:

    protected function copySkeleton(string $basePath, ?string $skeletonPath): void
    {
        if ($skeletonPath === null) {
            return;
        }

        if (is_dir($skeletonPath)) {
            // $skeletonPath : D:\vhosts\example.local/content-blocks-skeleton/content-element
            // Hotfix Windows
            $skeletonPath = GeneralUtility::fixWindowsFilePath($skeletonPath);
            // $skeletonPath : D:/vhosts/example.local/content-blocks-skeleton/content-element
            GeneralUtility::copyDirectory($skeletonPath, $basePath);
        }
    }
```

**Comment by nhovratov** [comment]
Ah, thank you. I will add this fix for windows for you.

**Comment by steffenmaechtel** [comment]
Thank you @nhovratov.

( Created the issue with wrong Account, it was me )

**Comment by nhovratov** [comment]
@steffenmaechtel Fixed in 1.1.1
Could you test it please and verify the fix works?

**Comment by steffenmaechtel** [comment]
@nhovratov tested now with 1.1.1

It works :) Thank you
